src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -458,9 +448,7 @@
                 def {@methodName(method)}(
                         self,
                         options=None):
-                    @if documentation
-                        {@documentation}
-                    @end
+                    {@methodComments(methodView, methodView.doc)}
```

Comment:
I'm a little surprised this works without the `if` guard; typically leaving out the guard causes an empty return value to render as a newline. Do we have a test case where the method comments are empty to confirm?